### ISSUE TITLE
Not really a bug, but it is possible to remove extensions.

### ISSUE THREAD

**Issue by FunnyDoggs** [bypassiwastaken/skiovox-helper#122]
I once saw an old issue about being unable to uninstall extensions, but I found if you click the puzzle icon next to the url bar past your pinned extensions, then click the three dots next to the extension you want removed, then click, "Remove from Chrome," confirm the action, and it will be uninstalled. This is the only method I have that works.

**Comment by Hydrax123445** [comment]
yeah. or you can just kill the extension but the easy way is to just remove it by doing what you said
